indico/modules/events/contributions/util.py

```diff
@@ -226,6 +232,11 @@ def import_contributions_from_csv(event, f):
         if not title:
             raise UserValueError(_("Row {}: contribution title is required").format(num_row))
 
+        
```

Comment:
Why so much space around everything? The only place a Python file should have two consecutive empty lines is between top-level functions/classes.